api/rds.proto

```diff
@@ -502,6 +511,15 @@ message VirtualHost {
   // precedence rules apply.
   repeated HeaderValueOption request_headers_to_add = 7;
```

Comment:
Why don't we have `request_headers_to_remove`?

Reply:
I don't have a use case for it at the moment. Not sure if was intentionally omitted before or not.

Reply:
OK. I think we will have a use case for this at Google, but we may also have a more sophisticated filter doing header transforms, so it doesn't have to be part of this `RouteConfiguration`.